### ISSUE TITLE
Tabs on edit collection page missing role context

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6799]
### Descriptive summary
On dev.nurax.samvera.org (Hyrax 5.0.1), the `Description`, `Branding`, `Discovery` and `Sharing` tabs on the `Edit Collection` page do not have the correct context role and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. As an admin, go to https://dev.nurax.samvera.org/dashboard/collections/bc386j44x/edit?locale=en and run the Site Improve browser extension

![roles_edit_collection](https://github.com/samvera/hyrax/assets/32644254/2fc8ee98-944c-4616-b5c0-a81313667fe7)

### Related Issues
#6797

**Comment by rjkati** [comment]
This issue is also present on the [edit file page (all tabs)](https://dev.nurax.samvera.org/concern/file_sets/p2676v71k/edit?locale=en). If it is preferable to create a separate issue for that instance, please comment here.

**Comment by christhepianist** [comment]
Confirmed that this is no longer an issue.
